### DISCUSSION TITLE
Multiples themes

### DISCUSSION THREAD

**Discussion by leandross2** [chakra-ui/chakra-ui#4589] [General]
Hi everybody!

Possible work with two or more themes with chakra?

In my project, the API set the style page, and would migrate to chakra

Someone help me with this?

Thanks!

Obs: my  front nextjs

**Comment by asffp** [comment]
Hello :) 
We did in our project with loading the theme config and passing it to ChakraProvider theme props

```


  const [theme,setTheme]=useState(undefined);

    useEffect(()=> {

        (async()=>{
         const response=  await  fetch('theme api ')
         const themeFromApi=   await response.json()
        setTheme(themeFromApi)
        })()
    }


     // in your JSX 
		    <ChakraProvider theme={theme}>
		    
				    your components that need chakra theme  
		    
		    </ChakraProvider>


```

**Reply by leandross2** [reply]
Nice!
I will try this way.
